tests/core/test_index.py

```diff
@@ -332,19 +328,28 @@ def test__supplement_index_with_prefix_index_class(
 
 def test__supplement_index_with_cache():
     idx = {}
-    _supplement_index_with_cache(idx)
+    with pytest.deprecated_call():
+        _supplement_index_with_cache(idx)
+        _supplement_index_with_cache(idx)
     tzdata = [p for p in idx.values() if p.name == "tzdata"][0]
     tzdata = PackageRecord.from_objects(tzdata)
     idx = {tzdata: tzdata}
-    _supplement_index_with_cache(idx)
+    with pytest.deprecated_call():
+        _supplement_index_with_cache(idx)
     augmented_tzdata = idx[tzdata]
     assert type(tzdata) is PackageRecord
     assert type(augmented_tzdata) is PackageCacheRecord
     assert tzdata == augmented_tzdata
 
 
 def test__make_virtual_package():
-    virtual_package = _make_virtual_package("name", "1.0", "0")
+    """
+    Ensures that the deprecated call and the new call are equivalent.
+
+    TODO: Remove this test when the deprecated call is removed.
```

Comment:
Totally fine with leave this here just FYI we don't need these comments, `pytest` will complain pretty loudly for us

Reply:
It's always nice to have a little more information though 😄